opengemini/write.go

```diff
@@ -84,17 +107,44 @@ func (c *client) WritePoint(database string, point *Point, callbackFunc func(err
 	req.queryValues.Add("db", database)
 	resp, err := c.executeHttpPost(UrlWrite, req)
 	if err != nil {
-		callbackFunc(err)
+		callback(err)
 	} else if resp.StatusCode != http.StatusNoContent {
 		var p []byte
 		_, err = resp.Body.Read(p)
 		if err != nil {
-			callbackFunc(errors.New("write failed ,status code:" + resp.Status + ",get resp body error for " + err.Error()))
+			callback(errors.New("write failed ,status code:" + resp.Status + ",get resp body error for " + err.Error()))
 		} else {
-			callbackFunc(errors.New(resp.Status + " :" + string(p)))
+			callback(errors.New(resp.Status + " :" + string(p)))
 		}
 	} else {
-		callbackFunc(nil)
+		callback(nil)
 	}
 	return nil
 }
+
+func (c *client) internalBatchSend(database string, resource <-chan *sendBatchWithCB) {
+	var tickInterval = time.Duration(c.config.BatchConfig.BatchInterval) * time.Second
+	var ticker = time.NewTicker(tickInterval)
+	var points = new(BatchPoints)
+	var cbs []WriteCallback
+	var waitFullQueue atomic.Bool
```

Comment:
This boolean variable name is a little confusing, we can use a better name like `needFlush`